packages/framework/pr_tools/trilinosprhelpers/TrilinosPRConfigurationStandard.py

```diff
@@ -18,6 +18,9 @@ class TrilinosPRConfigurationStandard(TrilinosPRConfigurationBase):
     """
     def __init__(self, args):
         super(TrilinosPRConfigurationStandard, self).__init__(args)
+        if "extra_configure_args" in args:
+            d = vars(args)
+            self._arg_extra_configure_args = d["extra_configure_args"]
```

Comment:
I don't think this is needed, the property code in the base class should take care of this.